cmd/fossa/cmd/upload_project/upload_project.go

```diff
@@ -0,0 +1,58 @@
+package upload_project
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/api/fossa"
+	"github.com/fossas/fossa-cli/cmd/fossa/display"
+	"github.com/fossas/fossa-cli/cmd/fossa/flags"
+	"github.com/fossas/fossa-cli/cmd/fossa/setup"
+	"github.com/fossas/fossa-cli/config"
+)
+
+var Cmd = cli.Command{
+	Name:      "upload-project",
+	Usage:     "Uploads source code to fossa for server side scanning",
```

Comment:
I would be more explicit: "Uploads a module's source code as a top-level FOSSA project rather than a dependency."